src/json-api/json-api-module.ts

```diff
@@ -394,7 +394,7 @@ export class JsonApiModule {
   }
 
   public toApiNotificationConfiguration(configuration: NotificationConfiguration): ApiNotificationConfiguration {
-    return configuration as ApiNotificationConfiguration;
+    return { ...configuration } as ApiNotificationConfiguration;
```

Comment:
If they have the same structure, shouldn't this work without the cast? Doesn't the `as ApiNotificationConfiguration` typecast hide errors in case the shapes change in the future?